jerry-main/main-unix.c

```diff
@@ -50,6 +50,25 @@
 
 static uint8_t buffer[ JERRY_BUFFER_SIZE ];
 
+#ifdef JERRY_PORT_ENABLE_EXTERNAL_CONTEXT
```

Comment:
I think main-unix is more like a demo for the default target, right? I put here to show the ability/ feasibility of the feature.
The writer of main-unix can decide the size of the jerry-heap during runtime, instead of during the jerry-core compiling time.

I also realize it may be not a proper place. I should put it inside a unit test case.